File: 1 - INICIANTE/python/1117.py
def notavalida(num):
    if num<0 or num>10:
        print('nota invalida')
    else:
        global cont, media
        media.append(num)
        cont += 1
        return num

media = list()
cont = 0

while True:
    num = float(input())
    notavalida(num)
    if cont == 2:
        break

print('media = {:.2f}'.format((media[0]+media[1])/2))

File: 1 - INICIANTE/1117.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;

int main () {

float a,b,x,y;
x = 0;
y = 0;

do {
  cin >> a;
  if (a>=0 && a<=10) {
    x = a;
  }
  else {
    cout << "nota invalida" << "\n";
  }
} while (a<0 || a>10);

do {
  cin >> b;
  if (b>=0 && b<=10) {
   y = b;
  }
  else {
    cout << "nota invalida" << "\n";
  }
} while (b<0 || b>10);

cout << "media = " << (a+b)/2 << "\n";

return 0;


}